Cargo.toml

```diff
@@ -16,24 +16,20 @@ exclude = [".github", "docker", ".gitignore", "Makefile"]
 [features]
 default = []
 native_tls = ["dep:pin-project", "dep:tokio-native-tls"]
-rustls = [
-    "dep:pin-project",
-    "dep:tokio-rustls",
-    "dep:rustls-native-certs",
-]
+rustls = ["dep:pin-project", "dep:tokio-rustls", "dep:rustls-platform-verifier"]
 binaries = ["dep:clap", "tokio/macros"]
 ent = []
 
 [dependencies]
 async-trait = "0.1.77"
 clap = { version = "4.4.10", optional = true }
-chrono = { version = "0.4", features = [
+chrono = { version = "0.4.20", features = [
```

Comment:
I think we were locked above this patch version, but still it's worth tightening due to potential [vulnerability](https://rustsec.org/advisories/RUSTSEC-2020-0159.html). Yay for `deps.rs` !

Reply:
In general, vulnerabilities shouldn't be patched in consumers through their `Cargo.toml` bounds — the lockfile is for this. The manifest (`Cargo.toml`) is _just_ for declaring bounds on what's _possible_ to build with. We don't want to restrict this to a newer version unnecessarily as it would mean consumers that are locked on an older version for some reason are stuck without a path forward.

Reply:
Hmm I agree that the bounds should be kept as low as possible, especially if the library is not relying on  features added in minors and patches of its dependencies. However, a vulnerability  fix may (should?) be considered a feature we are relying upon, so I am not sure this is an "unnecessary" bump. It somehow feels not right.

And also can they really get locked on an older version in this case? If they could build their app with `0.4.0` till `0.4.19`, they _should_ be able to build with `0.4.20`.

Reply:
No, a vulnerability fix is not something we depend on :)

And yes, they can, such as if they use `<=` or `=` dependency specifiers.

Reply:
this is technically possible, but I mean logically why would they go with `<=0.4.19` or `=0.4.19` for example :)

Reply:
For example because the newer version requires a newer version of Rust than what they can currently run (eg, due to OS limitations or internal company policies).